index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <title> How to view the HTML source of a Website on Chrome </title>
+    <link rel="stylesheet" href="./styles.css">
+  </head>
+
+  <body>
+<h1> How to view the HTML source of a Website on Chrome </h1>
+<img src='./img.png' alt='View Source Code picture' width="800" height="500">
+
+<h2> There are 4 steps you need to take in order to "reveal the magician's secrets"</h2>
+
+<ol>
+  <li>Open the browser (Google Chrome).</li>
+  <li>Go to the page you wish to see the HTML code for.</li>
+  <li>Right-click on the page.</li>
+  <li>Click 'View page source.'</li>
+</ol>
+
+<table style="width:100%">
+  <tr class='blue'>
```

Comment:
You can use the `:nth-child` pseudo-selector to select every even (or odd) numbered element. See https://www.geeksforgeeks.org/use-of-even-and-odd-pseudo-classes-with-list-items-in-css/

e.g., if you have `<table id="schedule">...</table>` you could do

```css
table#schedule tr:nth-child(even) {
  color: blue;
}

table#schedule tr:nth-child(odd) {
  color: red;
}
```